File: BackEnd/postman/CentroMedico.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='CentroMedico'
data={
  'accion':'crear',
  'nombre': None,
  'idTipoCentro': None,
  'direccion': None,
}
print postear(data,url)
print

url='CentroMedico'
data={
  'accion':'listarTipos',
}
print postear(data,url)
print

url='CentroMedico'
data={
  'accion':'listarTodos',
}
print postear(data,url)
print

url='CentroMedico'
data={
  'accion':'actualizar',
  'pnombre': None,
  'idTipoCentro': None,
  'idCentroMedico': None,
  'pdireccion': None,
}
print postear(data,url)
print

url='CentroMedico'
data={
  'accion':'listarPorTipo',
  'descripcion': None,
  'idTipoCentro': None,
}
print postear(data,url)
print

File: BackEnd/ajax/CentroMedico.js
$.ajax({
  url:SITIO_URL+'/services/CentroMedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'nombre': null,
    'direccion': null,
    'idTipoCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/CentroMedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTipos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/CentroMedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTodos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/CentroMedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idCentroMedico': null,
    'pnombre': null,
    'pdireccion': null,
    'idTipoCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/CentroMedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorTipo',
    'descripcion': null,
    'idTipoCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


